pymc/tests/distributions/test_distribution.py

```diff
@@ -0,0 +1,499 @@
+#   Copyright 2020 The PyMC Developers
```

Comment:
This one seems to have logic that makes more sense in other test files, I will leave specific suggestions sometime this week

Reply:
test_random and test_moments might be better of, split across discrete/continous/multivariate/... (whichever place the respective distributions are implemented in)

Reply:
Yes, I was indeed thinking to do random and moments next.